### PR TITLE
fix(DB/creature): Ulduar 25 extra trash packs

### PR DESCRIPTION
**Pull request by aradep**
## CHANGES PROPOSED:
In the 25-man version of Ulduar, there should be 2 packs of Iron Dwarf trash at the start of The Antechamber and at the base of the stairs leading to Kologarn.

## TESTS PERFORMED:
<details><summary>Before</summary>

image

image

</details>
<details><summary>After</summary>

image

image

</details>

## HOW TO TEST THE CHANGES:
Set raid difficulty 10-player
.go gob 34935
Confirm 1 pack of Iron Dwarf trash near teleporter and at the stairs leading to Kologarn.

Set raid difficulty 25-player
.go gob 34935
Confirm 2 packs of Iron Dwarf trash near teleporter and at the stairs leading to Kologarn.

## Target branch(es):
- [x] Master

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1605979091794346800.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    /* Iron Mender */
+    (136775, 34198, 603, 0, 0, 2, 1, 26218, 1, 1676.98, -11.2869, 427.31, 5.76657, 604800, 0, 0, 337025, 62535, 0, 0, 0, 0, '', 0);
+    
+/* Lightning Charged Iron Dwarf unsheath weapons and emote ready */
+DELETE FROM `creature_addon` WHERE `guid` IN (136609,136610,136771,136774);
+INSERT INTO `creature_addon` (`guid`, `bytes2`, `emote`) VALUES 
+('136609', '1', '333'),
+('136610', '1', '0'),
+('136771', '1', '0'),
+('136774', '1', '333');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1605979091794346800.sql:32]: int not string

Comment (aradep) [reply data/sql/updates/pending_db_world/rev_1605979091794346800.sql]: Good catch 👍

**Comment by Si1ker** [comment]
@aradep  source?

**Comment by Voxstrasza** [comment]
> 
> 
> @aradep source?

it's hard to find proof of this, but it is correct - there is extra trash in this hallway for the 25 man version.  streaming wasn't really a thing back then and there aren't any retail videos of a full ulduar 25 clear.

**Comment by Si1ker** [comment]
Found this one at 10:55
https://www.youtube.com/watch?v=m4fDOMYyjWw&ab_channel=AlValentyn

Seems this guy has been playing in retail since the first wow video in the channel so I'll take it as a base.
(ofc the feedback from Voxstrasza as well 🐱 )

**Comment by Voxstrasza** [comment]
> 
> 
> Found this one at 10:55.
> https://www.youtube.com/watch?v=m4fDOMYyjWw&ab_channel=AlValentyn
> In another xpac only, youtube does not have complete runs from that time ;c
> 
> Seems this guy has been playing in retail since the first wow video in the channel so I'll take it as a base.
> (ofc the feedback from Voxstrasza as well 🐱 )

I'll try to find a video from 2009/2010 for further proof if this doesn't suffice!

**Review by Si1ker — APPROVED** [review]
Tested ✔️ 

/* Affected rows: 10  Found rows: 0  Warnings: 0  Duration for 4 queries: 0.016 sec. */

### Before
image


image


### After
image

image

**Comment by FrancescoBorzi** [comment]
good job guys @aradep @Si1ker

**Pull request merged**